experiments/HSA_20150714_MI/hsa_20150714_MI.py

```diff
@@ -0,0 +1,286 @@
+        RackType='ITC Plate'))
+
+nreplicates = 1  # number of replicates of each experiment
+
+
+# #  Initial cleaning is skipped.
+# name = 'initial cleaning water titration'
+# itc_experiment_set.addExperiment(
+#     ITCExperiment(
+#         name=name,
+#         syringe_source=water_trough,
+#         cell_source=water_trough,
+#         protocol=cleaning_protocol,
+#         cell_volume=cell_volume))
+
+# Add water control titrations.
+for replicate in range(1):
+    name = 'water into water %d' % (replicate + 1)
+    itc_experiment_set.addExperiment(
+        ITCExperiment(
+            name=name,
+            syringe_source=water_trough,
+            cell_source=water_trough,
+            protocol=control_protocol,
+            cell_volume=cell_volume))
+
+# Add buffer control titrations.
+for replicate in range(1):
+    name = 'buffer into buffer %d' % (replicate + 1)
+    itc_experiment_set.addExperiment(
+        ITCExperiment(
+            name=name,
+            syringe_source=buffer_trough,
+            cell_source=buffer_trough,
+            protocol=blank_protocol,
+            cell_volume=cell_volume))
+
+## We didn't include this buffer into HSA titration in our experiment design, Is it necessary?
+
+# # buffer into hsa
+# for replicate in range(1):
+#     name = 'buffer into HSA %d' % (replicate + 1)
+#     itc_experiment_set.addExperiment(
+#         ITCExperiment(
+#             name=name,
+#             syringe_source=buffer_trough,
+#             cell_source=hsa_solution,
+#             protocol=blank_protocol,
+#             cell_concentration=0.040 * ureg.millimolar,
+#             buffer_source=buffer_trough,
+#             cell_volume=cell_volume))
+
+# drugs/HSA
+# scale cell concentration to fix necessary syringe concentrations
+
+cell_scaling = 1.
+for drug, drug_solution, drug_ka in zip(drugs, drug_solutions, drug_kas):
+
+    # We need to store the experiments before adding them to the set
+    drug_protein_experiments = list()
+    drug_buffer_experiments = list()
+
+    # Scaling factors per replicate
+    factors = list()
+
+
+    # Define drug to protein experiments.
+    for replicate in range(1):
+        name = '%s into HSA %d' % (drug.name, replicate + 1 )
+        experiment = ITCHeuristicExperiment(
+            name=name,
+            syringe_source=drug_solution,
+            cell_source=hsa_solution,
+            protocol=binding_protocol,
+            cell_concentration=0.040 *
+            ureg.millimolar *
+            cell_scaling,
+            buffer_source=buffer_trough,
+            cell_volume=cell_volume)
+        # optimize the syringe_concentration using heuristic equations and known binding constants
+        # TODO extract m, v and V0 from protocol somehow?
+
+        ## I am not sure what this part is doing?
+
+        # Warning, you're possibly not getting the setup you want. Consider not using the Heuristic Experiment
+        experiment.heuristic_syringe(drug_ka, 10, strict=False)
```

Comment:
This part tries to optimize the syringe concentrations based on a known/predicted Ka value. See the docstrings for ITCHeuristicExperiment.heuristic_syringe. I put the warning there because I set strict to False. When strict=False, it might mean the equation couldn't be fit within a reasonable range, so it uses a default concentration.